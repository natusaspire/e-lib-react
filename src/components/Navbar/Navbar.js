import React from 'react';
import { NavLink } from 'react-router-dom';

const Navbar = () => (
  <div className="navbar">
    <ul className="navbar__body">
      <li className="navbar__item navbar__item--first">
        <NavLink
          exact
          to="/"
          className="navbar__link"
          activeClassName="navbar__link--active"
        >
          Home
        </NavLink>
      </li>
      <li className="navbar__item">
        <NavLink
          exact
          to="/library"
          className="navbar__link"
          activeClassName="navbar__link--active"
        >
          Library
        </NavLink>
      </li>
    </ul>
  </div>
);

export default Navbar;
