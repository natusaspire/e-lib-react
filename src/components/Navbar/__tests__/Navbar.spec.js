import React from 'react';
import { shallow } from 'enzyme';

import Navbar from '@/components/Navbar';

jest.mock('react-router-dom', () => ({ NavLink: 'NavLink' }));

describe('<Navbar />', () => {
  it('should render navbar content', () => {
    const wrapper = shallow(<Navbar />);

    expect(wrapper).toMatchSnapshot();
  });
});
