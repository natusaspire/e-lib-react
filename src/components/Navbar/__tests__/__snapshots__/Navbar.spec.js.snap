// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render navbar content 1`] = `
<div
  className="navbar"
>
  <ul
    className="navbar__body"
  >
    <li
      className="navbar__item navbar__item--first"
    >
      <NavLink
        activeClassName="navbar__link--active"
        className="navbar__link"
        exact={true}
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li
      className="navbar__item"
    >
      <NavLink
        activeClassName="navbar__link--active"
        className="navbar__link"
        exact={true}
        to="/library"
      >
        Library
      </NavLink>
    </li>
  </ul>
</div>
`;
